CMakeLists.txt

```diff
@@ -74,5 +74,14 @@ IF(Trilinos_ENABLE_DataTransferKitMoabAdapters)
 ENDIF()
 
 ##---------------------------------------------------------------------------##
+## Check that there are no trailing whitespace
+##---------------------------------------------------------------------------##
+
+ADD_TEST(
+    NAME check_no_trailing_whitespace
+    COMMAND sh -c "grep --exclude=*.git* -rl '[[:blank:]]$' ${CMAKE_SOURCE_DIR} test $? -eq 1"
+)
```

Comment:
Would this not include the whole Trilinos? Even if you run it in DTK source, would that not include files like pdf if they are there? I think it should only be enforced for some file types, and not for all files.